include/gba/types.h

```diff
@@ -41,7 +41,7 @@ struct BgCnt
     u16 screenBaseBlock:5;
     u16 areaOverflowMode:1;
     u16 screenSize:2;
-};
+} PACKED;
```

Comment:
Based on your description, maybe this would work:
```suggestion
} ALIGNED(2);
```